src/state/pipeline-run-task/index.js

```diff
@@ -0,0 +1,7 @@
+import { makeLocalGetter } from '../../utils/object';
+import get from 'lodash/get';
+
+const localGetter = makeLocalGetter('pipelineRunTask');
+
+export const getPipelineRunTask = (state) =>
+  get(state, 'pipelineRunTask', null);
```

Comment:
remove unused getter

```suggestion
import get from 'lodash/get';

export const getPipelineRunTask = (state) =>
  get(state, 'pipelineRunTask', null);
```